reports/filesDataExporter/handlers.go

```diff
@@ -0,0 +1,172 @@
+package filesDataExporter
+
+import (
+	"bytes"
+	"context"
+	"sort"
+
+	"github.com/go-openapi/runtime"
+	"github.com/go-openapi/strfmt"
+	"github.com/rs/zerolog"
+
+	"github.com/iryonetwork/wwm/gen/storage/client/operations"
+	"github.com/iryonetwork/wwm/gen/storage/models"
+	"github.com/iryonetwork/wwm/reports"
+)
+
+type handlers struct {
+	source         *operations.Client
+	sourceAuth     runtime.ClientAuthInfoWriter
+	dataSanitizer  Sanitizer
+	reportsStorage reports.Storage
+	logger         zerolog.Logger
+}
+
+// ExportFile export new files and file updates to reports storage.
+func (h *handlers) ExportFile(ctx context.Context, bucketID, fileID, version string, timestamp strfmt.DateTime) (ExportResult, error) {
+	// Get file from source storage
+	var buf bytes.Buffer
+
+	getParams := operations.NewFileGetVersionParams().
+		WithBucket(bucketID).
+		WithFileID(fileID).
+		WithVersion(version).
+		WithContext(ctx)
+	_, err := h.source.FileGetVersion(getParams, h.sourceAuth, &buf)
```

Comment:
invalid operation: h.source (variable of type *invalid type) has no field or method FileGetVersion